### DISCUSSION TITLE
How to draw perfect horizontal or vertical lines?

### DISCUSSION THREAD

**Discussion by caronte3d** [pubpub-zz/ppInk#207] [Q&A] [answered]
I was try with Ctrl or Shift keys, but nothing seems to work

**Comment by pubpub-zz** [comment]
If you are using the line tool, without pressing any key the line will tool will 'suggest' to draw lines at 0, 15°,30°, 45°,...., (You need to keep cursor next to the wished direction

Without pressing, it may catch an edge, curve or the middle of a line. To prevent this extra magnet effect, press shift.

**Comment by caronte3d** [comment]
Not mi case, the line doesn't step at angles, only attach to  another objects

https://user-images.githubusercontent.com/4162639/189921719-7673b040-686e-4700-9f76-7d06298cc350.mov

**Comment by pubpub-zz** [comment, marked as answer]
Have you done any adjustment in the options. 
Can you send me the config.ini file ?

**Reply by caronte3d** [reply]
.ini renamed to .txt to enable github attach
[config.txt](https://github.com/pubpub-zz/ppInk/files/9577132/config.txt)

**Reply by pubpub-zz** [reply]
in your config: you have modified:
```
#Magnetic Angle Steps in deg
Magnetic_Angle = 450
```
This disables the globally the magnetic angle algorithm. if you want to just do vertical and horizontal set the value to 90 (in the file or in the options)

Also remember that the tool will propose to correct the arrow end only if you are +/-2° away from required direction

**Reply by caronte3d** [reply]
I don't remember to chante that value.
Now works nice.
Thank for your help.

**Reply by pubpub-zz** [reply]
You are welcomed. Do not hesitate to star the project if you enjoy it 😊

**Reply by caronte3d** [reply]
Done! ;)
